stock_by_warehouse/static/src/js/widget.js

```diff
@@ -6,27 +6,34 @@ var Widget = require('web.Widget');
 var form_common = require('web.AbstractField');
 var formats = require('web.field_utils');
 var fieldRegistry = require('web.field_registry');
+var {EventDispatcherMixin} = require('web.mixins');
 var QWeb = core.qweb;
+var _t = core._t;
 
-var Locations = Widget.extend({
-    init: function (info) {
+var Locations = Widget.extend(EventDispatcherMixin, {
+    init: function (parent, info) {
+        this._super(...arguments);
         this.info = info;
     },
 
     start: function () {
+        this._super(...arguments);
         var info = this.info;
         if (info !== false) {
             _.each(info.content, function (k, v) {
                 k.index = v;
-                k.name = k.warehouse_name
+                k.name = k.warehouse_name;
```

Comment:
Didn't this fail?

Reply:
Apparently no, I just find it, and it worked even before this change hahaha :see_no_evil: